laghos.cpp

```diff
@@ -61,6 +61,7 @@
 #include "laghos_rom.hpp"
 #include "laghos_utils.hpp"
 #include <fstream>
+#include <map>
```

Comment:
Can this be removed?

Reply:
Yes. The `unordered_map` call still works.